scripts/bootstrap.js

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env babel-node
+
+import inquirer from 'inquirer';
+import program from 'commander';
+import childProcess from 'child_process';
+
+const spawn = childProcess.spawnSync;
+
+program
+  .version('1.0.0')
+  .option('--all', 'Run all without asking')
```

Comment:
I think I agree ... just raised the consideration of keeping the normal bootstrap behaviour default with the interactive bootstrapping be a power mode.  But this is more discoverable!